tutorindigo/templates/indigo/lms/static/sass/home/_home.scss

```
@@ -131,3 +131,93 @@ section.home {
         }
     }
 }
+
+
+.home-container {
+    .slider-container {
+          position: relative;
+          overflow: hidden;
+          width: 90%;
+          margin: 0 auto;
+
+         .courses-listing.slider-track {
+          display: flex;
+          transition: transform 0.3s ease-in-out;
+          width: 98%;
+          margin: 0 auto;
+        }
+
+        .slider-card {
+          flex: 0 0 100%;
+          padding: 10px;
+          margin: 0 !important;
+        }
+
+        .indicators {
+          text-align: center;
+          margin-top: 10px;
+        }
+        .indicator {
+          display: inline-block;
+          width: 10px;
+          height: 10px;
+          background: $gray;
+          border-radius: 50%;
+          margin: 0 5px;
+          cursor: pointer;
+        }
+        .indicator.active {
+          background: $primary;
+        }
+
+        @media (min-width: 576px) {
+          .slider-card {
+            flex: 0 0 50%;
+          }
+        }
+
+        @media (min-width: 768px) {
+          .slider-card {
+            flex: 0 0 31.3333%;
+          }
+        }
+    }
+
+    section.courses {
+        position: relative;
+        margin: 1vw 0;
+    }
+    .nav-btn {
+      position: absolute;
+      top: 50%;
+      transform: translateY(-50%);
+      background: rgba(0, 0, 0, 0);
+      color: $gray;
+      border: none;
+      padding: 10px;
+      cursor: pointer;
+      z-index: 10;
+
+      &:hover, &:focus, &:active {
+        color: $primary;
+      }
+    }
```

I suggest add the following styles to override the default buttons ones

```suggestion
     font-size: 1.5rem;
     z-index: 10;
     box-shadow: none;

      &:hover, &:focus, &:active {
        color: $primary;
        background: none;
        box-shadow: none;
      }
    }
```

---

Thank you very much! 🙌